catalyst-gateway/bin/src/service/api/documents/post_document_index_query/response.rs

```diff
@@ -145,9 +184,49 @@ impl Example for IndexedDocumentVersion {
     }
 }
 
+/// Document Reference for filtered Documents.
+#[derive(NewType, Debug, Clone, From, Into)]
+#[oai(
+    from_multipart = false,
+    from_parameter = false,
+    to_header = false,
+    example = true
+)]
+pub(crate) struct FilteredDocumentReference(DocumentReference);
```

Comment:
Why we need an additional structure `FilteredDocumentReference` which is just a wrapper over `DocumentReference`. Is the `DocumentReference` already an OpenAPI type ?